float8_experimental/float8_ops.py

```diff
@@ -140,3 +141,29 @@ def autocast_to_copy(aten_op, args, kwargs=None):
     return Float8Tensor(
         args[0]._data, args[0]._scale, kwargs["dtype"], args[0]._emulate
     )
+
+
+@implements([c10d_functional.all_gather_into_tensor.default])
+def allgather_fp8(aten_op, args, kwargs=None):
+    """
+    override funcol with FP8 handling
+    """
+    fp8_input = args[0]
```

Comment:
nit: I am not familiar, but what if the user passes only via `kwargs`?
Similarly, do we need `if kwargs is None: kwargs = {}`?

Reply:
I think our python arg parser would parse the python inputs using the operator schema, so even if user pass as kwarg, it would be passed as arg if the operator does not have this kwarg.

For this kwarg handling
```
if kwargs is None: kwargs = {}
```
iirc passing kwarg directly down to the aten call should not have a problem, I think some of the someclasses (i.e. DTensor) handle kwarg this way mainly because of avoiding type annotation and isinstance checks of None values.